### ISSUE TITLE
training-data sample

### ISSUE THREAD

**Issue by mrocr** [opconty/pytorch_ctpn#8]
@opconty 
What type of labels are required for the training-data?
Can you provide a sample of the training data?

Waiting for your reply

**Comment by kiiokiio** [comment]
我觉得应该是对每一行的宽度按照16进行划分label的标注

**Comment by 980044579** [comment]
You can modify the corresponding [data reading module](https://github.com/opconty/pytorch_ctpn/blob/master/data/dataset.py#L66) according to your actual data format.
